rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -134,6 +135,63 @@ void fmt_class_string<CellSysutilParamId>::format(std::string& out, u64 arg)
 	});
 }
 
+s32 SysutilIsValidNameString(const char* src, s32 minlen, s32 maxlen)
```

Comment:
Is its name re-ed or something? Otherwise I'd suggest changing naming stile to_lowercase_underscored.

Reply:
alright then, it's name is not re-ed.